Changed 404 and landing page (staging)

justmedev opened the pull request:
Added some messages for more clearance

crispyaus committed:
Create LICENSE

crispyaus committed:
only deploy when master branch is pushed

crispyaus committed:
rename

crispyaus committed:
Merge pull request #12 from dogegarden/staging

Staging

crispyaus committed:
Merge pull request #14 from dogegarden/staging

fixed avatarURL

crispyaus committed:
Merge pull request #15 from dogegarden/staging

Update App.js

justmedev committed:
Changed 404 and landing

Added messages for more information

crispyaus reviewed (changes requested):
dogehouse.tv not dogehouse.com, also maybe provide link to docs on the 404

justmedev committed:
Update App.js

justmedev commented:
> 
> 
> dogehouse.tv not dogehouse.com, also maybe provide link to docs on the 404

Done. Changed 404 to also provide a link to the gitbook API docs and changed my typo for landing.
